.env.test

```diff
@@ -1,12 +1,10 @@
-# Used for image placeholder functionality
-NEXT_PUBLIC_DPL_CMS_HOSTNAME=route-to-dpl-cms.com
-
 # DPL-CMS
-NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_DPL_CMS=https://route-to-dpl-cms.com/graphql
+NEXT_PUBLIC_DPL_CMS_HOSTNAME=http://localhost:9000
+NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_DPL_CMS=http://localhost:9000/graphql
 NEXT_PUBLIC_GRAPHQL_BASIC_TOKEN_DPL_CMS=Z3JhcGhxbF9jb25zdW1lcjp0ZXN0
 
 # FBI API
-NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_FBI=https://temp.fbi-api.dbc.dk/ereolgo/graphql
+NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_FBI=http://localhost:9000/graphql
```

Comment:
II think it would be nice to write a commet saying that localhost:9000 is our mock server. Setting the value here basically means that whenever we run some test in CI this will be used. Which is fine - but also nice to know :)